extensions/wikia/Recirculation/js/recirculation.js

```diff
@@ -52,12 +67,15 @@ define('ext.wikia.recirculation.recirculation', [
 				});
 				break;
 			case 'VIDEOS_MODULE':
-			default:
 				videosModule(element);
 				break;
+			default:
+				videosModule(element);
+				return;
 		}
 
-		$(element).on('click', 'a', trackClick);
+		trackImpression();
+		$(element).on('mousedown', 'a', trackClick);
```

Comment:
Mousedown is more reliable when the link that is clicked sends the user to an external page. See: https://github.com/Wikia/app/blob/dev/resources/wikia/modules/tracker.js#L320